tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -434,20 +437,75 @@ func (iptConfigurator *IptablesConfigurator) run() {
 	iptConfigurator.handleInboundIpv4Rules(ipv4RangesInclude)
 	iptConfigurator.handleInboundIpv6Rules(ipv6RangesExclude, ipv6RangesInclude)
 
-	// Execute iptables commands
-	for _, cmd := range iptConfigurator.iptables.BuildV4() {
+	iptConfigurator.executeCommands()
+}
+
+func (iptConfigurator *IptablesConfigurator) createRulesFile(f *os.File, contents string) error {
+	defer f.Close()
+	writer := bufio.NewWriter(f)
+	_, err := writer.WriteString(contents)
+	if err != nil {
+		fmt.Println("Unable to write file for iptables-restore")
```

Comment:
I'd prefer not to write into standard output if possible, to make this code more reusable.
Maybe just:
```suggestion
		return fmt.Errorf("failed to write iptables-restore file: %v", err)
```

Reply:
done